saleor/discount/models.py

```diff
@@ -149,12 +149,16 @@ def validate_min_amount_spent(self, value):
 
 
 class SaleQueryset(models.QuerySet):
-    def active(self, date):
+    def active(self, date=None):
+        if date is None:
+            date = timezone.now().date()
```

Comment:
Start and End date in Sale model are DateTimeField we should use 
`timezone.now()` instead of `timezone.now().date()`